client/src/main/java/com/alibaba/nacos/client/security/SecurityProxy.java

```diff
@@ -41,45 +43,45 @@
  * @since 1.2.0
  */
 public class SecurityProxy {
-    
+
```

Comment:
Do not change the indent .
Please use nacos code style to reformat code.

detail see soruce code `style/CodeStyle.md`

Reply:
sorry
I have modified and re-uploaded